auto-dark.el

```diff
@@ -178,22 +219,45 @@ already set the theme for the current dark mode state."
     (unless (eq appearance auto-dark--last-dark-mode-state)
       (auto-dark--set-theme appearance))))
 
+(defun auto-dark--update-frame-backgrounds (appearance)
```

Comment:
Really nice! 

Thanks for providing both functionalities for now.

Idn why we didn't use semver for this package in the beginning.

Maybe it's time for it? What do you think of we call this changes version 0.12.1? 

Then make-obsolete-variable would point to 0.13.0.

I was thinking about making a migration guide right now. But new docs + old functionality for now may be enough.

Reply:
Yeah, I’m happy to do whatever you want here. I certainly am in favor of SemVer (https://github.com/sellout/emacs-semver/ [^1]).

[^1]: Hmmm, it looks like I need to push a newer version of that library.

> What do you think of we call this changes version 0.12.1?
> Then make-obsolete-variable would point to 0.13.0.

I’m a little confused with this part. Are you suggesting that #56 gets tagged 0.12.1, and then this PR is tagged as 0.13.0? `make-obsolete-variable` should include whatever version we add the function call in, not a future version. And I think both #56 and this are probably equivalently minor version bumps?

A migration guide would be nice, but the warning when the code is byte-compiled does look like
```
example.el:1:1: Warning: ‘auto-dark-light-theme’ is an obsolete variable
    (as of 0.14); use ‘auto-dark-themes’ instead.
```
which isn’t the worst warning message I’ve seen.

Reply:
Hmm nice, never used this feature before.

Yeah, I was suggesting this one to become 0.12.1 and leave 0.13.0 to the future (when the old method would become obsolete).

I see it. Never used make-obsolete-variable before, If it needs the 'current version', let's give it to it.

I see no problems calling "this version" 0.13.0, if you could please bump the version number on the top of the file, it would be nice.